src/components/DAG/components/DAGContent.tsx

```diff
@@ -83,12 +83,14 @@ const DAGBranch: React.FC<DAGBranchProps> = ({ steps, structure, goToStep }) =>
           // Because its actually previous step that contains info about the split, we need
           // to check it here.
           const previousStep = index === 0 ? null : structure[index - 1];
-          const containerType = typeof previousStep === 'string' ? steps[previousStep].type : 'split';
+          const containerType = typeof previousStep === 'string' ? steps[previousStep].type : 'split-static';
 
           return (
             <DAGContainerItem
               key={index}
-              type={containerType === 'foreach' || containerType === 'split' ? containerType : 'split'}
+              type={
+                containerType === 'split-foreach' || containerType === 'split-static' ? containerType : 'split-static'
```

Comment:
We could change this to `['split-foreach', 'split-parallel'].includes(containerType) ? containerType : 'split-static'`. Little bit cleaner and more correct. We probably need to fix some styling for split-parallel later.